content/en/docs/sdk-utilities/s3.md

```diff
@@ -0,0 +1,430 @@
+	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+var (
+	localPath string
+	bucket    string
+	prefix    string
+)
+
+func init() {
+	if len(os.Args) != 4 {
+		log.Fatalln("Usage:", os.Args[0], "<local path> <bucket> <prefix>")
+	}
+	localPath = os.Args[1]
+	bucket = os.Args[2]
+	prefix = os.Args[3]
+}
+
+func main() {
+	walker := make(fileWalk)
+	go func() {
+		// Gather the files to upload by walking the path recursively 
+		if err := filepath.Walk(localPath, walker.Walk); err != nil {
+			log.Fatalln("Walk failed:", err)
+		}
+		close(walker)
+	}()
+
+	cfg, err := config.LoadDefaultConfig()
+	if err != nil {
+		log.Fatalln("error:", err)
+	}
+	
+	// For each file found walking, upload it to S3
+	uploader := manager.NewUploader(s3.NewFromConfig(cfg))
+	for path := range walker {
+		rel, err := filepath.Rel(localPath, path)
+		if err != nil {
+			log.Fatalln("Unable to get relative path:", path, err)
+		}
+		file, err := os.Open(path)
+		if err != nil {
+			log.Println("Failed opening file", path, err)
+			continue
+		}
+		defer file.Close()
+		result, err := uploader.Upload(context.Background(), &s3.PutObjectInput{
+			Bucket: &bucket,
+			Key:    aws.String(filepath.Join(prefix, rel)),
+			Body:   file,
+		})
+		if err != nil {
+			log.Fatalln("Failed to upload", path, err)
+		}
+		log.Println("Uploaded", path, result.Location)
+	}
+}
+
+type fileWalk chan string
+
+func (f fileWalk) Walk(path string, info os.FileInfo, err error) error {
+	if err != nil {
+		return err
+	}
+	if !info.IsDir() {
+		f <- path
+	}
+	return nil
+}
+```
+
+## Download Manager
+
+The {{% alias service="S3" %}} [Downloader]({{% apiref "feature/s3/manager#Downloader" %}}) manager determines if a file
+can be split into smaller parts and downloaded in parallel. You can customize the number of parallel downloads and the
+size of the downloaded parts.
+
+#### Example: Download a File
+
+The following example uses the {{% alias service="S3" %}} `Downloader` to download a file. Using `Downloader` is similar
+to the [s3.GetObject]({% apiref "service/s3#Client.GetObject" %}}) operation.
+
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+import "github.com/aws/aws-sdk-go-v2/service/s3"
+import "github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	log.Println("error:", err)
+	return
+}
+
+client := s3.NewFromConfig(cfg)
+
+downloader := manager.NewDownloader(client)
+numBytes, err := downloader.Download(context.Background(), downloadFile, &s3.GetObjectInput{
+	Bucket: aws.String("my-bucket"), 
+	Key:    aws.String("my-key"),
+})
+```
+
+The `downloadFile` parameter is an `io.WriterAt` type. The `WriterAt` interface enables the `Downloader` to write
+multiple parts of the file in parallel.
+
+#### Configuration Options
+
+When you instantiate a `Downloader` instance, you can specify several configuration options to customize how objects are
```

Comment:
Think "several" might be overstating things here; looks like there are only two. Could just omit "several."